components/export/api/src/main/java/org/phenotips/export/internal/DataToCellConverter.java

```diff
@@ -247,35 +246,53 @@ public DataSection phenotypeBody(Patient patient) throws Exception
         if (sortedFeatures.size() == 0) {
             // offset is included to account for the presence of both "positive" and "negative" in "present"
             int offset = bothTypes ? 1 : 0;
-            Integer emptyX = 0;
             for (int i = 0; i < present.size() - offset; i++) {
-                DataCell cell = new DataCell("", emptyX, 0);
+                DataCell cell = new DataCell("", i, 0);
                 section.addCell(cell);
-                emptyX++;
             }
         }
         // section.finalizeToMatrix();
         return section;
     }
 
+    public void variantsSetup(Set<String> enabledFields) throws Exception
+    {
```

Comment:
That's fair. :) In the future, I think it would be important to lay out those assumptions and prerequisites at the top of the PR then, and maybe suggest people wait to review it until after variants is merged and it has been rebased?